sale_order_import_edifact/wizard/sale_order_import.py

```diff
@@ -0,0 +1,249 @@
+import json
+import logging
+import os
+from base64 import b64decode
+from collections import defaultdict
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+logger = logging.getLogger(__name__)
+
+
+class SaleOrderImport(models.TransientModel):
+    _inherit = "sale.order.import"
+
+    import_type = fields.Selection(
+        selection_add=[("edifact", "EDIFACT")], ondelete={"edifact": "cascade"}
+    )
+    state = fields.Selection(
+        selection_add=[("tech", "Technical")], ondelete={"tech": "set default"}
```

Comment:
explanation on this "state"?

Reply:
It's just to show the object parsed for debug. 
We can remove the state and show the `parsed_json` field only if it contains data.

Reply:
I'd rather add a specific field or yes. check that you have a value there